File: Cap 04/4.3 - Desvio Condicional Simples/adcnum1.kt
/*
 * Livro .....: Algoritmos
 *              Lógica para Desenvolvimento de Programação de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Tópico 4.3
 * 
 * Programa ..: ADICAO DE NUMEROS - DECISAO SIMPLES
 * 
 */

import java.util.Scanner

fun main() {

    var entrada = Scanner(System.`in`)
    var A: Float
    var B: Float
    var X: Float

    println("DECISAO SIMPLES - ADICAO DE DOIS NUMEROS\n")

    print("Entre varor <A>: ")
    A = entrada.nextFloat()

    print("Entre varor <B>: ")
    B = entrada.nextFloat()

    X = A + B

    if (X > 10) {
        print("Resultado da adicao quando maior que dez = ")
        System.out.printf("%.2f", X)
        println()
    }
}

File: Cap 04/4.3 - Desvio Condicional Simples/adcnum1.vb
' Livro .....: Algorimos
'              Logica para Desenvolvimento de Programacao de 
'              Computadores
' Autores ...: Jose Augusto N. G. Manzano
'              Jayr Figueiredo de Oliveira
' Editora ...: Erica
' 
' Capitulo ..: 4 - Topico 4.Programacao
' 
' Programa ..: ADICAO DE NUMEROS - DECISAO SIMPLES

Module AdicaoDeNumeros

    Sub Main()

        Dim A As Single, B As Single, X As Single
        
        Console.WriteLine("DECISAO SIMPLES - ADICAO DE DOIS NUMEROS")
        Console.WriteLine()

        Console.Write("Entre valor <A>: ")
        A = Console.ReadLine()

        Console.Write("Entre valor <B>: ")
        B = Console.ReadLine()

        X = A + B

        If (X > 10) Then
            Console.WriteLine("Resultado da adicao quando maior que dez = " & X.ToString("###.##"))
        End If

    End Sub

End Module
